configure.ac

```diff
@@ -5404,7 +5433,9 @@ int main(void)
 [ac_cv_buggy_getaddrinfo=no],
 [ac_cv_buggy_getaddrinfo=yes],
 [
-if test "${enable_ipv6+set}" = set; then
+if test "$ac_sys_system" = "Linux-android" || test "$ac_sys_system" = "iOS"; then
+  ac_cv_buggy_getaddrinfo="no"
```

Comment:
+1 to this for the iOS case; I've used --enable-ipv6 for all my builds, and it works fine. I see no reason it shouldn't be enabled by default.